libs/ledger-live-common/src/sanction/index.ts

```diff
@@ -0,0 +1,59 @@
+import { LiveConfig } from "@ledgerhq/live-config/LiveConfig";
+import axios from "axios";
+import { hours, makeLRUCache } from "@ledgerhq/live-network/cache";
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { getCurrencyConfiguration, getSharedConfiguration } from "../config";
+import { CurrencyConfig } from "@ledgerhq/coin-framework/lib/config";
+
+const cache = makeLRUCache(fetchSanctionedAddresses, () => "all_sanctioned_addresses", hours(12));
+
+async function fetchSanctionedAddresses(): Promise<Record<string, string[]>> {
+  const { data } = await axios.get(
+    `https://ofac-compliance.pages.dev/all_sanctioned_addresses.json`,
+  );
+  return data;
+}
```

Comment:
```suggestion
async function fetchSanctionedAddresses(): Promise<Record<string, string[]>> {
  const { data } = await axios.get(
    "https://ofac-compliance.pages.dev/all_sanctioned_addresses.json",
  );
  return data;
}
```
Question: do you think `@ledgerhq/live-network` is suitable here ? It comes with cache and retry policies

Reply:
You can keep the cache as is. But instead of `axios`, you could use
```typescript
import network from "@ledgerhq/live-network";
```
The `network` function is a thin wrapper around `axios` that comes with retry capabilities.

Reply:
I will make this change in another PR, we need this for a common part with @qperrot